docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "2.4"
```

Comment:
Where does this version number come from?

Reply:
This is the version of the docker-compose file format, the latest is 3 but not really adapted to test/demo docker

Reply:
I am not familiar with docker-compose. Has directory /app no relation to docker's WORKDIR?